.github/workflows/deploy-to-aks.yaml

```diff
@@ -31,13 +32,30 @@ jobs:
       - name: Process bundle versions
         id: process_bundle_versions
         run: python scripts/gh-actions/parse_versions.py "${{ inputs.bundle_version }}"
-          
+
+  generate-bundle-versions-on-schedule:
+    if: ${{ github.event_name == 'schedule' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install CLI tools
+        run: |
+          sudo snap install yq
+        
+      - name: Create JSON array of bundle versions
+        run: |
+          TARGET_VERSIONS="1.8, 1.9, latest"
+          bundle_versions=$(yq '. | keys | map(select(. == env(TARGET_VERSIONS).split(", ")[]))' .github/dependencies.yaml -o=json)
```

Comment:
I think instead of introducing TARGET_VERSIONS, we 'd be better off removing the [1.7 entry](https://github.com/canonical/bundle-kubeflow/blob/main/.github/dependencies.yaml#L1). This is a file solely used in the CI and since 1.7 is out of support, we don't need to keep it there. WDYT? FYI we had discussed this [here](https://chat.canonical.com/canonical/pl/nn76p4e8obrxzedgansyfdeita) too.

Reply:
I missed that message, adjusting right now!